[2023-12-27 17:54:33,855] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD manual__2023-12-27T17:54:20.503605+00:00 [queued]>
[2023-12-27 17:54:33,881] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD manual__2023-12-27T17:54:20.503605+00:00 [queued]>
[2023-12-27 17:54:33,882] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 17:54:33,883] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-27 17:54:33,884] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 17:54:33,925] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-12-27 17:54:20.503605+00:00
[2023-12-27 17:54:33,938] {standard_task_runner.py:52} INFO - Started process 13067 to run task
[2023-12-27 17:54:33,948] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'conexion_BD', 'manual__2023-12-27T17:54:20.503605+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmps15v9lbc', '--error-file', '/tmp/tmp625gj6x0']
[2023-12-27 17:54:33,951] {standard_task_runner.py:80} INFO - Job 99: Subtask conexion_BD
[2023-12-27 17:54:34,191] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.conexion_BD manual__2023-12-27T17:54:20.503605+00:00 [running]> on host 44a36bef83ad
[2023-12-27 17:54:34,300] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-27 17:54:34,436] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T17:54:20.503605+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T17:54:20.503605+00:00
[2023-12-27 17:54:34,439] {logging_mixin.py:115} INFO - Conectandose a la DB en la fecha: 2023-12-27 17
[2023-12-27 17:54:35,672] {logging_mixin.py:115} INFO - <connection object at 0x7fd719cdc440; dsn: 'user=juliocfurtado93_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-12-27 17:54:35,673] {logging_mixin.py:115} INFO - Conexion a RedShift exitosa!
[2023-12-27 17:54:35,675] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 17:54:35,724] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=conexion_BD, execution_date=20231227T175420, start_date=20231227T175433, end_date=20231227T175435
[2023-12-27 17:54:35,811] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 17:54:35,965] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
